uk.co.compucorp.civicrm.hrcontactactionsmenu/js/contactactions.js

```diff
@@ -1,17 +1,31 @@
-(function ($, CRM) {
-  $(document)
-    // Actions menu
-    .on('click', function (e) {
-      if ($(e.target).is('#crm-contact-actions-link, #crm-contact-actions-link *')) {
-        $('#crm-contact-actions-link').addClass('active');
-      } else {
-        $('#crm-contact-actions-link').removeClass('active');
-      }
+(function (CRM) {
+  var $ = CRM.$;
+
+  (function init () {
+    initListeners();
+  }());
+
+  /**
+   * Contains event listeners
+   */
+  function initListeners () {
+    // Toogle Actions Menu's 'active' class
+    $(document).on('click', function (e) {
+      toggleActionMenuButtonStatus(e);
+    });
+
+    // Handler for deleting user account
+    $('[data-delete-user-url]').on('click', function () {
+      confirmUserAccountDeletion(this);
```

Comment:
It is a bit strange to see `this` passed as a param. May be use `bind` instead?

Reply:
I have refactored the code.